### PR TITLE
ADR list and ADR views

### PR DESCRIPTION
**Pull request by PGaigole**
- ADRs list sorted in desc order.
- ADR listing linked to respective ADR content.
- ADR list made filterable based on category.
- Metadata on ADR view.

### CODE REVIEW

File: assets/adr.scss [hunk 1]
```diff
@@ -1,57 +1,93 @@
...
-    .status-proposed {
-      background-color: #2196f3; /* Blue */
+    .adr-status {
+      padding: 5px 10px;
+      border-radius: 3px;
+      font-size: 0.9em;
+      margin-bottom: 10px;
+      float: right;
+      .status-accepted {
+        background-color: #4caf50; /* Green */
```
Comment (hussainweb) [inline assets/adr.scss:38]: Let's use variables for these as well.

File: assets/js/filter-adr.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  var adrCards = document.querySelectorAll('.adr-card');
+  var filterButtons = document.querySelectorAll('.filter-button');
+
+  adrCards.forEach(function (card) {
+    var categories = card.getAttribute('data-categories').split(' ');
+    if (category === 'All' || categories.includes(category)) {
+      card.style.display = 'block';
+    } else {
+      card.style.display = 'none';
+    }
```
Comment (hussainweb) [inline assets/js/filter-adr.js:11]: How do you feel about replacing this with the ternary operator?

```suggestion
    card.style.display = (category === 'All' || categories.includes(category)) ? 'block' : 'none';
```

File: assets/js/filter-adr.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  filterButtons.forEach(function (button) {
+    var buttonText = button.textContent.trim(); // Trim whitespace
+    if (buttonText === category) {
+      button.classList.add('active');
+    } else if (category === 'All' && buttonText === 'All') {
+      button.classList.add('active');
+    } else {
+      button.classList.remove('active');
+    }
```
Comment (hussainweb) [inline assets/js/filter-adr.js:22]:
```suggestion
    button.classList.remove('active');
    if (buttonText === category) {
      button.classList.add('active');
    }
```

**Review by hussainweb — COMMENTED** [review]
Copying over some of my comments from #191 and couple fresh ones.

**Review by hussainweb — APPROVED** [review]

File: assets/js/filter-adr.js (hunk 2 above)
Comment (PGaigole) [reply assets/js/filter-adr.js]: @hussainweb Done

File: assets/js/filter-adr.js (hunk 3 above)
Comment (PGaigole) [reply assets/js/filter-adr.js]: @hussainweb done

**Pull request merged**